### DISCUSSION TITLE
Headed arbitrary class can't be extracted

### DISCUSSION THREAD

**Discussion by thuanthe81** [tailwindlabs/tailwindcss#6905] [Help] [answered]
tailwindcss: 3.0.8
postcss-cli 9.1.0
Node: v16.13.1
Browser: Chrome
OS: ubuntu 20

Description:
  Arbitrry class can't can't be extracted with: `<div class="py-[20.125rem]"></div>`
  However it can be extracted success with: `<div class=" py-[20.125rem]"></div>`

**Comment by reinink** [comment]
Hey there!

I've just tested both of these situations in Tailwind Play, and it's working for me: https://play.tailwindcss.com/iRmwebrz4m

Without a minimal reproduction repo it's very difficult for us to help. Feel free to provide one, and I'd be happy to have another look 👍

**Reply by thuanthe81** [reply]
I just checked with html file, it's not problem, too. Sorry the issue is different with my above description.

Here is repository demo above issue:
    `https://github.com/thuanthe81/shadow-cljs-tailwindcss/tree/6890-headed-arbitrary`
 
It's clojurescript project. After build cljs -> js, I build css, but don't see `px-[5.125rem]` and `w-[1.123rem]` in output css file.
But when I add space before above class `" w-[1.123rem]" ` `" w-[1.123rem]"`, they will be in output css file.

**Comment by thuanthe81** [comment, marked as answer]
I resolved this. Thanks.
